org.ultimatepp.TheIDE.metainfo.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+    <id>org.ultimatepp.TheIDE</id>
+    <launchable type="desktop-id">org.ultimatepp.TheIDE.desktop</launchable>
```

Comment:
@bbhtt and @hfiguiere is launchable OK or it should be changed? If it should be changed please let me know to what?

Reply:
looks correct now